src/lua/LuaPlayer.cpp

```diff
@@ -686,6 +687,167 @@ static int l_player_is_hyperspace_active(lua_State *l)
 	return 1;
 }
 
+/*
+ * Function: GetCruiseDirection()
+ *
+ * Returns:
+ *
+ *   string - 'CRUISE_UP' or 'CRUISE_FWD'
+ *
+ */
+static int l_player_get_cruise_direction(lua_State *l)
+{
+	Player *player = LuaObject<Player>::CheckFromLua(1);
+	auto mode = player->GetPlayerController()->GetCruiseDirection();
+	auto str = EnumStrings::GetString("CruiseDirection", mode);
+	assert(str && "Wrong get cruise direction");
```

Comment:
Instead of using `assert` in these functions, please use `luaL_error()` instead - it's nicer error handling and it's one less program crash I have to remove when making Lua code hot-reloadable and fault tolerant.

Reply:
Done! By the way, these assertions helped me a lot earlier.